daemon/inertiad/project/data.go

```diff
@@ -138,6 +139,14 @@ func (c *DeploymentDataManager) GetEnvVariables(decrypt bool) ([]string, error)
 	return envs, err
 }
 
+// AddBuiltContainer stores and tracks metadata from successful builds
+func (c *DeploymentDataManager) AddBuiltContainer(commitHash string, containerID string) error {
```

Comment:
for expandability sake I might make the value some kind of struct, so that we can add more fields to it in the future

for example:

```go
Container{ ID string }
```

and then we can have a signature like:

```suggestion
func (c *DeploymentDataManager) AddBuiltContainer(hash string, containers ...Container) error {
```

there `containers` utilizes the spread operator to allow an arbitrary number of `Container` (remember that docker-compose builds will have multiple containers, when we get around to supporting that)

that way, if we want to add build duration, you can update `Container`:

```go
Container{ ID string, BuildDuration int64 }
```